meetings/2022-12-14.md

```diff
@@ -69,11 +94,55 @@
+* PA: My point was the scope of each specification might be too narrow to justify use cases about that specification. I would feel more comfortable with one use case document maybe not for whole Solid but for core portions. There is a [use case document](https://www.w3.org/TR/rdf-dawg-uc/) from RDF Access working group that produced SPARQL. They don't point to specific recommendations they produced, but they have one document for use cases that covers a number of specifications.
+* SC: Good point. I don't think we have that. The user stories come close to that as a collection of things. As you can see, only two or three attempts at having UCR documents in context of authorization and notifications. Anything from Solid Protocol wasn't though through more than institutional knowledge, derived from lessons learned from LDP.  Are you suggesting we should have one overarching use case document? For all work items in Solid ecosystem?
+* PA: Yeah, it's a gut feeling, it would make more sense.
+* WT: I think I second that. We do have two big files with use cases now but even in those, there are subdivisions into sub-specs, so why not make one file of it and have different sections and label accordingly. I think that would be nice.
+* SC: We could. Accidental bottom-up that we have general need for a component like authorizations or notifications, some spec is needed focused only on that. You can take a requirement like creating a container vice listening to updates to a container, they could go in same use case document, the approach here was that they are specific enough so you could have different solutions, so notifications could be a solution to listening and leave that in its own document. Since this is somewhat bottom-up, we could merge or create a document that touches these different areas. 
+* WT: Don't stop on account of me. I wanted to say to give the example about notifications, a use case involving notifications is ever completed without also some implementation of authentication and authorization. So people from authorization have to know to look there as well that involve authorization but are not in our use cases, so if in one document, easier to find.
+* SC: Good clarification. Thank you for raising that. We should have a closer look at how we make it more obvious and we could have See Also's, but I suppose in case of authorization, you would need use cases around notifications or considerations of how spec would need to advise while direct access may not be enabled through the spec, information can be shared in other ways, implementors may need to be aware. A spec could expose info not intended by authorization spec. Something we need to discuss in panels as well. 
```

Comment:
```suggestion
* SC: Good clarification. Thank you for raising that. We should have a closer look at how we make it more obvious. We could have `SeeAlso`s, but I suppose in case of authorization, you would need use cases around notifications or considerations of how spec would need to advise while direct access may not be enabled through the spec, information can be shared in other ways, implementors may need to be aware. A spec could expose info not intended by authorization spec. Something we need to discuss in panels as well. 
```